### DISCUSSION TITLE
Consider non-grouped styles as `base`

### DISCUSSION THREAD

**Discussion by hennessyevan** [macaron-css/macaron#29] [Ideas]
In Stitches, we put "base" styles in the object like so
```ts
const RedBox = styled('div', {
  backgroundColor: '$red'
})
```
In macaron, following vanilla-extract, we use the `base` keyword
```ts
const RedBox = styled('div', {
  base: {
    backgroundColor: '$red'
  }
})
```

I'm not sure what the benefit of using `base` is here, perhaps there is some good reason but could we possibly support both syntaxes? It would surely make it just a little easier for our team to migrate from stitches to macaron.

**Comment by Mokshit06** [comment]
Hey, so the main reason for having `base` here is that it allows you to use pre-defined styles/classes as the base styles for some element, so you can do

```js
const RedBox = styled('div', {
    base: "some-class-name",
})
```

We could support both syntaxes though

**Reply by hennessyevan** [reply]
Oh that's cool 👍 

When we get to it, I'll see whether we might just do a codemod and if we do, I'll contribute that codemod here of course
